sponsorship_compassion/models/res_partner.py

```diff
@@ -416,19 +436,18 @@ def _random_str():
                 "category_id": [(5, 0, 0)],
                 "comment": False,
                 "active": False,
+                "gender": False,
+                "title": False,
+                "zip": False,
+                "city": False,
+                "country_id": False,
+                "currency_id": False,
+                "gmc_gender": False,
+                "phone_sanitized": False,
+                "short_address": False,
+                "social_sec_nr": False,
```

Comment:
Be careful, if you access fields that are not declared in this module or a parent module, this code will fail when the module creating this field is not installed. The `social_sec_nr` is a nordic-specific field which we won't have in switzerland. We should therefore not use this one in this code. The correct way of doing it is to inherit the `forget_me` function inside the compassion-nordic repository for handling all Nordic custom fields. Please review also the other fields you added, to make sure their use is valid here.
```suggestion
```